modules/custom/openy_gtranslate/src/Plugin/Block/OpenYGTranslateBlock.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Drupal\openy_gtranslate\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * Provides a 'OpenY Google Translate' block.
+ *
+ * @Block(
+ *   id = "openy_gtranslate_block",
+ *   admin_label = @Translation("OpenY Google Translate"),
+ *   category = @Translation("OpenY"),
+ * )
+ */
+class OpenYGTranslateBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $block = [
+      '#theme' => 'openy_gtranslate',
+      '#cache' => ['max-age' => 0],
```

Comment:
Why remove caching? Could this block be cached?
It is on every page, I'm not a big fan of adding not cached blocks for all pages